versions/3.0.3.md

```diff
@@ -2326,7 +2326,7 @@ Other than the JSON Schema subset fields, the following fields MAY be used for f
 ##### Fixed Fields
 Field Name | Type | Description
 ---|:---:|---
-<a name="schemaNullable"></a>nullable | `boolean` | Allows sending a `null` value for the defined schema. Default value is `false`.
+<a name="schemaNullable"></a>nullable | `boolean` | A `true` value adds `"null"` to the allowed type specified by the `type` keyword, only if `type` is explicitly defined within the same Schema Object. Other Schema Object constraints retain their defined behavior, and therefore may disallow the use of `null` as a value. A `false` value leaves the specified or default `type` unmodified. The default value is `false`.
```

Comment:
Do we also need to mention and clear up that type being missing is string, integer, null, array, etc?

Reply:
> Do we also need to mention and clear up that type being missing is string, integer, null, array, etc?

I think that's covered by JSON Schema. `type` is a constraint, and in the absence of a `type` constraint, values of any type are allowed. 

I would not mind stating it explicitly, but TSC folks on the call were pushing to keep the spec minimal and concise. Paraphrasing what was said on the call, people will inevitably get confused and ask questions, and no amount of explanatory verbiage in the spec will prevent that from happening.

Reply:
@philsturgeon we covered this in the call.  We want to keep the text minimal, and defer to JSON Schema as much as possible.